testing/security/transport_encryption.py

```
@@ -6,11 +6,63 @@
 
 
 import sdk_cmd
-
+import sdk_security
+import sdk_utils
 
 log = logging.getLogger(__name__)
 
 
+def setup_service_account(service_name: str,
+                          service_account_secret: str=None) -> dict:
+    """
+    Setup the service account for TLS
+    """
+
+    if sdk_utils.is_open_dcos():
+        log.warn("The setup of a service account requires DC/OS EE")
```

I'd throw an exception here and crash the test.